### PR TITLE
[DISTROS] Adds support for Nobara

### PR DESCRIPTION
**Pull request by Snootiful**
## Description

This PR adds ASCII art for the Nobara Project.
(The ASCII art was not made by me, but using an ASCII art generator. I would like to encourage someone who has experience making ASCII art to improve this icon.)

I have not merged the branch I created with the master branch. As this PR should not cause merge conflicts I will leave the merging to whoever accepts this PR.

Also, the provided screenshots show warnings about temperature sensors. This warning also appears without my changes, because this is an issue with my system. The warnings are not related to the changes this PR makes.

## Reason and / or context

This PR was created to support the Nobara Project. The Red Hat logo would show without this PR.

## How has this been tested ?

Environment:
Tested this PR on my machine running Nobara 37 using both the fish and bash shell. Should work on other machines too since the code in this PR matches that of other PRs that add logos for various distros.

archey-fish

archey-bash

Built using PEX:

archey-build-pex

## Types of changes :

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Typo / style fix (non-breaking change which improves readability)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist :

- [ ] \[IF NEEDED\] I have updated the _README.md_ file accordingly ;
- [ ] \[IF NEEDED\] I have updated the test cases (which pass) accordingly ;
- [ ] \[IF BREAKING\] This pull request targets next Archey version branch ;
- [X] My changes looks good ;
- [X] I agree that my code may be modified in the future ;
- [X] My code follows the code style of this project ([PEP8](https://www.python.org/dev/peps/pep-0008/)).

### CODE REVIEW

File: archey/logos/nobara.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""Nobara logo"""
+
+from archey.colors import Colors
+
+COLORS = [Colors.WHITE_BRIGHT]
+
+# pylint: disable=line-too-long
+LOGO = [
+    """{c[0]}  .cONWMWNOl;cdkKXWWMMMMWWXKkdc'.         """,
```
Comment (HorlogeSkynet) [inline archey/logos/nobara.py:9]: Hi @Snootiful ! Thanks for this very complete PR.

What about removing one (or both) leading whitespace(s) for this new logo ?

```suggestion
    """{c[0]} .cONWMWNOl;cdkKXWWMMMMWWXKkdc'.         """,
```

Thanks, bye 👋

File: archey/logos/nobara.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"""Nobara logo"""
+
+from archey.colors import Colors
+
+COLORS = [Colors.WHITE_BRIGHT]
+
+# pylint: disable=line-too-long
```
Comment (HorlogeSkynet) [inline archey/logos/nobara.py:7]: (I don't think we need this Pylint flag here, this logo is pretty "thin" 😉)

**Comment by Snootiful** [comment]
Thanks for the quick reply :)

The reason I added the Pylint flag and the leading whitespace was because I used the Red Hat logo as reference. I also don't have Pylint installed because I don't often program in Python.

I have removed one space from each line of the logo and removed the Pylint flag as suggested.

**Comment by Snootiful** [comment]
Oh, and I just noticed that I left a trailing comma on the LOGO array, which doesn't follow the code style from PEP8.
My apologies, feel free to remove that comma.

**Comment by HorlogeSkynet** [comment]
> Oh, and I just noticed that I left a trailing comma on the LOGO array, which doesn't follow the code style from PEP8.

Don't worry it's actually OK. We use Black code formatter over here, which makes sure there are trailing commas for multiple lines object definitions.

**Pull request merged**